torch/fx/experimental/proxy_tensor.py

```diff
@@ -410,3 +410,47 @@ def wrap_fake(x):
         return t
 
     return wrapped
+
+
+def _find_proxy_tensor(*objects_to_search):
+    # return first proxy tensor found or None
+    proxy_tensors = (o for o in objects_to_search if isinstance(o, ProxyTensor))
+    return next(proxy_tensors, None)
+
+
+def get_isolated_graphmodule(func, args, kwargs):
```

Comment:
Would we want some special decomposition here to short-cut things like `nvfuser.var_mean`? With a proper `impl_nvfuser` this short-cut would work out well :tada: